src/styles/skeleton.ts

```diff
@@ -0,0 +1,49 @@
+import { css, unsafeCSS } from 'lit';
+
+export default (container: string, bone: string) => {
```

Comment:
These make-shift mixins of ours are actually pretty flexible. If we later add another type of skeleton, we can either rename this one to "loading" and have two mixins or else change this one's API:

```ts
${skeleton({ 
  selectors: { 
    container: string, 
    bone: string,
  }, 
  type?:  'simple' | 'advanced' = 'simple'
})}
```

Reply:
"bone" comes from Design, by the way.

Reply:
🦴